milestone-delivery/deliveries/miden.md

```diff
@@ -0,0 +1,14 @@
+# Milestone Delivery :mailbox:
+
+> Only the GitHub account, which is responsible for the pull request of the accepted application is allowed to submit milestones. Don't remove any of the mandatory parts presented in bold letters or as headlines!
+
+**The Ethereum address has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](../).**  
+
+* **PR Link:** https://github.com/vechain/grant-program/pull/135 
+* **Milestone Number:** 5
+
+Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which should be used for testing.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Smart Contract & Backend |0xa1ada849b79a71529e61615de5c32dfe86b95b7e4547b4dd26bdb011e4c614cd | ...| 
```

Comment:
Seems incomplete, please kindly check.

Reply:
@libotony This was updated with all the milestones and deliverables completed. See commit with hash 3319bf89d26e68ecf414930182c78434f98f0f15

Reply:
@stratisnik Thanks for the update, could you please take a look at my previous comment regarding the application file?